README.md

```diff
@@ -1,12 +1,74 @@
-# About
-GitHub Action to execute tests by Testing Farm and update Pull Request status.
+# testing-farm-as-github-action
 
-The Action uses ubuntu-20.04
+Testing Farm as GitHub Action is a GitHub Action to execute tests by Testing Farm and update Pull Request status.
```

Comment:
```suggestion
Testing Farm as GitHub Action is a GitHub Action to execute tests by Testing Farm and update Pull Request status.
```
Testing Farm as a GitHub Action is a GitHub Action for executing tests against specific Pull Requests on the Testing Farm infrastructure. The GitHub action reports the result of the tests with Pull Request status.